pkg/reconciler/instances/istio/istio_test.go

```diff
@@ -292,3 +300,16 @@ func newFakeKubeClient() *k8smocks.Client {
 
 	return mockClient
 }
+
+type TestCommanderResolver struct {
```

Comment:
Why isn't this struct generated by `mockery`? That's our convention to create mocks this way.

Reply:
Let's leave it for later.